static/explorer.css

```diff
@@ -154,3 +154,59 @@ pre.content {
     width: 5px !important;
     left: 3px;
 }
+
+.thinking {
+    float:right;
+    display:inline;
+}
+
+/* http://bootsnipp.com/snippets/featured/glyphicon-animate-rotation-and-flip */
```

Comment:
I got this rotation animations from a snippet. It is fairly standard code so I dont feel necesary that this stays. But thanks tho!